cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

```diff
@@ -6,12 +6,16 @@ assigned to your user account or a team to which you belong.
 To assign permissions, you need access to Access Management and the Organization Administrator permission.
 
 * To set up and manage a private space, you need the CloudHub Network Administrator permission.
-* To view a private space, you need the CloudHub Network Viewer permission.
-** To view the schedules in applications deployed to CloudHub 2.0 and Runtime Fabric, you must assign the Exchange Viewer permission to the Business Organization that owns the asset.
+* To view a private space as an individual user within an organization, you need at least one of the following permissions:
+** Cloudhub Network Viewer
```

Comment:
Can you add what each of this permissions correspond to? 
** Cloudhub Network Viewer - Will be able to view the private spaces and it's details
** Organization Administrator - Will be able to manage and update private spaces.
** Cloudhub Network Administrator - Will be able to manage and update private spaces.